test/go_test.rb

```diff
@@ -80,6 +80,28 @@ def test_capture_three_stones
       assert_equal 3, game.captures[:black]
     end
 
+    def test_snack_back
+      game = Game.new(board: 9)
+      game.black(3,2)
+      game.black(2,3)
+      game.black(4,3)
+      game.black(2,4)
+      game.black(5,4)
+      game.black(3,5)
+      game.black(4,5)
+      game.white(4,2)
+      game.white(5,3)
+      game.white(3,4)
+      game.white(4,4)
+      game.white(2,0)
```

Comment:
What's this about?

Reply:
This test case is the same as below:
https://ja.wikipedia.org/wiki/%E3%82%A6%E3%83%83%E3%83%86%E3%82%AC%E3%82%A8%E3%82%B7

Reply:
Yeah I'm just wondering about the move at
White(0, 2)

Reply:
Just a mistake.
Removed.